pycytominer/cyto_utils/features.py

```diff
@@ -154,7 +154,7 @@ def drop_outlier_features(
         Features present in the population dataframe. If "infer", then assume Cell Painting features are those that start with "Cells_", "Nuclei_", or "Cytoplasm_"
     samples : list of str or str, default "all"
         Samples to perform the operation on
-    outlier_cutoff : int or float, default 15
+    outlier_cutoff : int or float, default 500
```

Comment:
```suggestion
    outlier_cutoff : int or float, default 500
    see https://github.com/cytomining/pycytominer/issues/237 for details.
```

Adding this explicit pointer for potential future reference